### PR TITLE
Logo106

### LINKED ISSUE #106: Logo banner
Screenshot 2023-06-26 at 10 37 57 AM

The image/logo in the banner is clickable and leads to another webpage. 

As seen in https://sgcreative.southernglazers.com/bloominbrands/p/2

Leverage a section with a style metadata (eg. partner-banner) and autoblocking.

### PR DESCRIPTION
**Pull request by atopper**
## Issue

Fix #106 

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/drafts/top/column-centered
- After: https://logo106--sgws-creative--hlxsites.hlx.page/drafts/top/column-centered

Original: https://sgcreative.southernglazers.com/bloominbrands/p/2

Authoring: 
First table in link below is the implemented one.  The second suggestion was another idea (simple section).  The last one was a old implementation.
https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/doc.aspx?sourcedoc=%7Be4a56646-8bd9-4f1f-82d6-e237d80b2095%7D&action=edit&cid=f21c1f7a-e0bd-473d-ad59-95d9e2b31a24

## 📝 Description:
Show a logo, some text and link them to an url.
Took suggestion 1 (banner table, 1 row of image & text/url).

### CODE REVIEW

File: blocks/columns/columns.js [hunk 1]
```diff
@@ -15,6 +27,26 @@ export default function decorate(block) {
...
+    const bannerCols = [...firstRow.children];
+    if (bannerCols.length === 2) {
+      processCellPicture(bannerCols[0], 'animate');
+
+      const linkP = bannerCols[1].querySelector('p:last-child');
+      const bannerLink = linkP.innerText;
+      bannerCols[1].removeChild(linkP);
+
+      firstRow.addEventListener('click', (e) => {
+        e.stopPropagation();
```
Comment (tmathern) [inline blocks/columns/columns.js:42]: Is this really needed? What are the side-effect?

**Review by tmathern — CHANGES_REQUESTED** [review]

**Review by tmathern — APPROVED** [review]

**Review by badvision — APPROVED** [review]
This makes sense, thank you!

File: blocks/columns/columns.js (hunk 1 above)
Comment (atopper) [reply blocks/columns/columns.js:44]: There is no concrete reason - I saw it elsewhere.  But I suppose they could put a banner over some larger, clickable image or something... maybe?  And we wouldn't want the click to propagate to the lower element.  That's what I assumed it was being used for.

File: blocks/columns/columns.js [hunk 2]
```diff
@@ -6,6 +6,20 @@ import {
...
+    const picWrapper = pic.closest('div');
+    if (picWrapper && picWrapper.children.length === 1) {
+      // picture is only content in column
+      picWrapper.classList.add('columns-img-col');
+      if (imgClassName) {
+        picWrapper.classList.add(imgClassName.trim());
+      }
+    }
+  }
+}
```
Comment (atopper) [inline blocks/columns/columns.js:21]: Hey - this shouldn't be here anymore.  I cleaned it up, but it must have reverted back.
Shoot...  I update it.

**Pull request merged**